### ISSUE TITLE
building wheel for fastparquet ... error

### ISSUE THREAD

**Issue by extreme4all** [dask/fastparquet#492]
python version:
`Python 3.8.2`
command:
`pip install fastparquet`

error: 
`Building wheels for collected packages: fastparquet
  Building wheel for fastparquet (setup.py) ... error`

`
building 'fastparquet.speedups' extension
  error: Microsoft Visual C++ 14.0 is required. Get it with "Build Tools for Visual Studio": https://visualstudio.microsoft.com/downloads/
  ----------------------------------------
  ERROR: Failed building wheel for fastparquet
`

i tried https://www.scivision.dev/python-windows-visual-c-14-required/

any suggestions? would love to use this tool as parquet seems to be fast and efficient way of storing data

**Comment by martindurant** [comment]
Indeed the build dependencies of fastparquet are not well specified, because it has binary extensions, and many people install via conda for this very reason. 
The module in question is cython code, and you will need a full compiler toolchain available on your system for the build to succeed.

**Comment by extreme4all** [comment]
i guess it is not as simple as 
`pip install Cython`

**Comment by martindurant** [comment]
No, sorry. That just installs the cython library, but if you're not using conda, the compilers and system libraries must be installed by you (or a sysadmin)

**Comment by extreme4all** [comment]
I'm not using conda, which compilers and system libraries must I install?
would be nice if you can list them & where to find them.

**Comment by martindurant** [comment]
I don't know for windows :|  You might want to check what conda-forge uses as it's build environment or generally search for cython/numpy/windows setup.

That is why conda has proven so helpful to people...
